pkgs/development/compilers/llvm/common/bolt/default.nix

```diff
@@ -83,13 +84,20 @@ stdenv.mkDerivation (finalAttrs: {
   postInstall = ''
     mkdir -p $dev/lib
     mv $out/lib/libLLVMBOLT*.a $dev/lib
+
+    mkdir -p $out/nix-support
+    cp $setupHook $out/nix-support/setup-hook
```

Comment:
Doesn't `mkDerivation` do this?

Reply:
Idk, I couldn't get the hook to appear for whatever reason.